### PR TITLE
Feat/release copy

### PR DESCRIPTION
**Pull request by frankmarra**
This PR adds the [react-quill](https://www.npmjs.com/package/react-quill#custom-formats) package. This is implemented in create release and update release. It allows pro subscribers to add an about section to a release.

### CODE REVIEW

File: components/InputReleaseAbout/InputReleaseAbout.jsx [hunk 1]
```diff
@@ -0,0 +1,39 @@
...
+import "react-quill/dist/quill.snow.css"
+
+const QuillNoSSRWrapper = dynamic(import("react-quill"), {
+  ssr: false,
+  loading: () => <p>Loading...</p>,
+})
+
+const modules = {
+  toolbar: [
+    [{ header: "2" }],
```
Comment (hexagoncircle) [inline components/InputReleaseAbout/InputReleaseAbout.jsx]: Remove this. I don't think folks need the ability to add headers if this is just a blurb.

File: components/Releases/ReleaseLayout.jsx [hunk 2]
```diff
@@ -56,6 +56,10 @@ export default function ReleaseLayout({
           isSubscribed={isSubscribed}
           isDlcmFriend={isDlcmFriend}
         />
+        <section
+          className={styles.about}
+          dangerouslySetInnerHTML={{ __html: release.about }}
```
Comment (hexagoncircle) [inline components/Releases/ReleaseLayout.jsx]: I would _highly suggest_ you ensure this input is sanitized. Is there another way quill recommends you handle this?

The prop is called `dangerouslySetInnerHTML` for a reason. It's fine to use if you _know_ that the content is clean, but when it's coming from users, you run the risk of XSS and other security issues.

Comment (frankmarra) [reply components/Releases/ReleaseLayout.jsx]: added the [dompurify](https://github.com/cure53/DOMPurify) package to handle this.

File: components/InputReleaseAbout/InputReleaseAbout.jsx (hunk 1 above)
Comment (frankmarra) [reply components/InputReleaseAbout/InputReleaseAbout.jsx]: Removed, and also added alignment options.

**Pull request merged**